packages/host/app/components/operator-mode/card-schema-editor.gts

```diff
@@ -49,6 +51,7 @@ interface Signature {
     allowAddingFields: boolean;
     childFields: string[];
     parentFields: string[];
+    openDefinition: (moduleHref: string, codeRef: ResolvedCodeRef) => void;
```

Comment:
I updated openDefinition as an action that is passed bcos schema editor and inspector/detail panel both use it